sdk/core/logger/review/logger.api.md

```diff
@@ -0,0 +1,41 @@
+## API Report File for "@azure/logger"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import debug from 'debug';
+
+// Warning: (ae-forgotten-export) The symbol "AzureDebugger" needs to be exported by the entry point logger.d.ts
+// 
+// @public (undocumented)
+export const AzureLogger: AzureDebugger;
+
+// @public (undocumented)
```

Comment:
Can you log an issue to document all the undocumented stuff in this package?

Reply:
AzureLogger should be documented. The rest isn't exposed so documenting seems lower pri, but could be done.

Reply:
Added docs!